ansible/VERSION

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Will the versions be changed based on what is the latest on dockerhub

Reply:
In this change you would have to update that file manually to trigger a new version. The only automated process would be the "latest" tag being added on a merge to master. I wasn't sure how to do automated semantic versioning.
My other thought is to let the container versions follow the release schedule of the shared libraries. That way we could say everything on version x has been tested together and in between releases update the patch version if necessary. In the latter the patch version would be updated based on dockerhub.

Reply:
I think that makes sense to get some consistency with the libraries and infra but may be very difficult to maintain. My initial question was around more how does this reflect in regards to the versioning we already have out on dockerhub for ansible-executor and linchpin-executor? Does this reset the version?

Reply:
I've bumped the version numbers so they won't conflict with the existing image versions.